### ISSUE TITLE
how to implement the Question only baseline?

### ISSUE THREAD

**Issue by Guanyu-Lin** [leroy102/REVIVE#11]
Could you please share this baseline code?

**Comment by yuanze-lin** [comment]
Thank you for your attention, I have been busy with other ongoing projects, if you mean you want to train the model only with question-answer pairs, there are two options: 
        (1) You can simply modify the dataloader to remove context-aware prompts.
        (2) You can use original OK-VQA dataset for training.

**Comment by Guanyu-Lin** [comment]
Thanks for your kind reply, I simply remove the context-aware prompts in dataloader and run the test script. But I get an accuracy result at 51.82%, it seems it does not as what you report in the paper and beats many baselines. So I am wondering why it is so strong, maybe it is because the model is already well fine-tune?

**Comment by yuanze-lin** [comment]
Which experiment do you mean?

**Comment by yuanze-lin** [comment]
I've forgotten a lot of details, I just remember the accuracy of my previous baseline may be about 49% ~ 50%.

**Comment by yuanze-lin** [comment]
I've forgotten a lot of details about the baseline trained without using any prompts and visual features, but I think the accuracy of the previous baseline is not so high, which is similar to [KAT](https://arxiv.org/pdf/2112.08614.pdf). Our architectures are all built upon [FID](https://github.com/facebookresearch/FiD).

**Comment by yuanze-lin** [comment]
> Thanks for your kind reply, I simply remove the context-aware prompts in dataloader and run the test script. But I get an accuracy result at 51.82%, it seems it does not as what you report in the paper and beats many baselines. So I am wondering why it is so strong, maybe it is because the model is already well fine-tune?

Did you train the model from scratch using only the question-answer pairs, or you just use the provided model for inference without context-aware prompts? The accuracy of the baseline without using prompts trained from scratch shouldn't be so high.

If you want to implement your mentioned question-only baseline, which hasn't been shown in the paper, you should train the model from scratch with only question-answer pairs, I think the accuracy is just about 20%~30%.

**Comment by Guanyu-Lin** [comment]
> > Thanks for your kind reply, I simply remove the context-aware prompts in dataloader and run the test script. But I get an accuracy result at 51.82%, it seems it does not as what you report in the paper and beats many baselines. So I am wondering why it is so strong, maybe it is because the model is already well fine-tune?
> 
> Did you train the model from scratch using only the question-answer pairs, or you just use the provided model for inference without context-aware prompts? The accuracy of the baseline without using prompts trained from scratch shouldn't be so high.
> 
> If you want to implement your mentioned question-only baseline, which hasn't been shown in the paper, you should train the model from scratch with only question-answer pairs, I think the accuracy is just about 20%~30%.

Thanks for your reply, I also think so.

**Comment by yuanze-lin** [comment]
> > > Thanks for your kind reply, I simply remove the context-aware prompts in dataloader and run the test script. But I get an accuracy result at 51.82%, it seems it does not as what you report in the paper and beats many baselines. So I am wondering why it is so strong, maybe it is because the model is already well fine-tune?
> > 
> > 
> > Did you train the model from scratch using only the question-answer pairs, or you just use the provided model for inference without context-aware prompts? The accuracy of the baseline without using prompts trained from scratch shouldn't be so high.
> > If you want to implement your mentioned question-only baseline, which hasn't been shown in the paper, you should train the model from scratch with only question-answer pairs, I think the accuracy is just about 20%~30%.
> 
> Thanks for your reply, I also think so.

You're welcome : )
